src/main/kotlin/mate/academy/figure/Figure.kt

```diff
@@ -0,0 +1,80 @@
+package mate.academy.figure
+
+import mate.academy.model.Color
+import mate.academy.service.AreaOfFigureSupplier
+import mate.academy.service.DrawSupplier
+
+sealed class Figure(val color: Color): DrawSupplier {
+}
+
+class Circle(color: Color,

```

Comment:
Please fix code formatting.
You may use CTRL+ALT+L hotkey on Windows or similar on Mac